### DISCUSSION TITLE
zh/guide/drivers/quark

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#20] [Announcements]
# zh/guide/drivers/quark

AList v3的文档

https://alist.nn.ci/zh/guide/drivers/quark.html

**Comment by Tenkkk** [comment]
夸克开了会员，webdav下载，仍旧限速2M

**Reply by qingmuhy** [reply]
同样的问题，夸克这个限速太蛋疼了。我是限速到1M。

**Reply by 13323688772** [reply]
是超级会员 限速100K 正在找原因中，

**Comment by choateyang** [comment]
v3.2.0版本，夸克盘的Cookie无法填入

**Reply by lingyu0628** [reply]
那看一下是不是拓展拦截了

**Reply by choateyang** [reply]
确实，被adguard拦截了。

**Comment by iniekai** [comment]
3版本以后，这个方法就失效了

**Reply by tagwu0227** [reply]
那现在该怎么弄

**Reply by iliyz** [reply]
还是可以的，只是cookie换了地方，不在collect开头那个文件，可以在其他文件找，比如list

![list](https://imgs.iliy.eu.org/file/e4562296ccb4b7338cb13.jpg)
而且只是一段时间有效，失效要更新cookie

**Reply by Ari3126** [reply]
还是在sort里，一直往下翻，翻到请求标头那里，直接显示的就是cookie

**Reply by ghost** [reply]
yep

**Reply by sinaamos** [reply]
@iliyz
所有要cookie的网盘挂载，过期了都得重新设置一遍？

**Comment by gmshiwoge** [comment]
开通夸克会员下载速度没有提升

**Comment by BeyondWei** [comment]
linux上挂载结束以后，其他应用的docker容器启动使用挂载目录时显示没有权限

**Comment by ghost** [comment]
去哪找cookies？？？

**Reply by Ari3126** [reply]
还是在sort里，一直往下翻，翻到请求标头那里，直接显示的就是cookie

**Reply by ghost** [reply]
对

**Comment by HansO-o** [comment]
浏览器下载是限速2M的，只有客户端才可以不限速，感觉cookie应该是不一样的，能不能抓包出来客户端的cookie用呢？安卓的最好，因为安卓端的视频下载也不限速，其他客户端视频下载限速，其他文件下载不限速

**Reply by qingmuhy** [reply]
我试着抓取了iOS的cookie，但在填入后貌似没有作用，可能登录方式不一样需要作者适配吧。

**Reply by air0451** [reply]
哪咋整，别的网盘都能跑满带宽，就差他了

**Reply by ghost** [reply]
跟cookie有啥关系，cookie存储的不是账户信息吗，你说的是user-agent吧

**Reply by liaolia333** [reply]
对就是user-agent

**Comment by air0451** [comment]
下载速度常年维持在13M左右，已经是超级会员了。看来真的是网页版cookie的问题，现在急需移动端的cookie啊！哪位大神会！！！！

**Reply by qingmuhy** [reply]
![屏幕截图 2022-12-10 190152.png](https://s2.loli.net/2022/12/10/8poGaw6HJr2ETnk.png)
那你还是很快的啊，我一直都是这个速度。

**Reply by qingmuhy** [reply]
我是普通会员，看来和会员还是有关的啊。13M是13Mbps还是13Mbyte/s啊，如果是后者我觉得够用了啊。

**Reply by air0451** [reply]
是不是会员都是十来兆

**Reply by liaolia333** [reply]
你说的是user-agent

**Comment by Ari3126** [comment]
兄弟，你这。。





山野都有雾灯
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "alist-org/docs"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年12月10日(星期六) 晚上7:03
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [alist-org/docs] zh/guide/drivers/quark (Discussion #20)





 
我是普通会员，看来和会员还是有关的啊。13M是13Mbps还是13Mbyte/s啊，如果是后者我觉得够用了啊。

**Comment by LinK12312312** [comment]
我想我知道了一个好方法，对于大文件，使用idm或ndm进行多线程下载，我宽带300m可以跑满

**Reply by ghost** [reply]
那玩意？原理是多线程
他同时启动N个线程去下，宏观层次就有了高速度

**Comment by ZYALBERT** [comment]
今天才发现挂载夸克限速2M/s  一直以为是4K播放的格式问题 作者可以解决一下这个问题吗

**Comment by FopFat** [comment]
夸克网盘 从今天 早上 .网页内看视频.就一直提示重连.求解. V3.6版本.

**Reply by ghost** [reply]
不懂（（（AList很久没用了

**Comment by ChaXxl** [comment]
我这个 folder id 是 0 咋整
![](https://article.biliimg.com/bfs/article/b9f0d4e4199bae76ea16d64b2f5485dc18bed631.png)

**Reply by DamienMM** [reply]
同样的现象

**Reply by FeranyDev** [reply]
根目录id就是0呀有什么问题

**Reply by aaabbbccc100** [reply]
同样的现象

**Reply by liaolia333** [reply]
根目录啊我晕

**Comment by darknesscho** [comment]
今天下载的docker镜像,夸克用这种方式挂载不上了,提示:
Failed init storage but storage is already created: failed init storage: Get "https://drive.quark.cn/1/clouddrive/config?fr=pc&pr=ucpro": dial tcp: lookup drive.quark.cn: Try again

**Reply by darknesscho** [reply]
嗯?重启了alist容器后又可以了,不知道什么情况.

**Comment by dragonyao1984** [comment]
夸克限速2M

**Comment by 123xsz** [comment]
夸克的问题就是太慢...

**Comment by air0451** [comment]
夸克网页版要是不限速就完美了，现在阿里云盘不支持磁力链接是一大遗憾

**Comment by aurorax-neo** [comment]
挂载后可以看到文件 但是为啥无法下载

**Comment by kuko7** [comment]
请问对于夸克的限速现在有什么好的解决办法吗

**Comment by KeshawnZhen** [comment]
作为夸克svip，客户端播放视频的网速也不到1.5M。4K画质的视频，根本就放不了

**Comment by DamienMM** [comment]
ID现在是和谐了吗？现在就是0

**Comment by DamienMM** [comment]
了解，第一天连不上，后来连上了，我以为是那个的问题

在 2023-02-26 11:11:36，FeranyDev ***@***.***> 写道：


根目录id就是0呀有什么问题

**Comment by Saber1u** [comment]
我以为只有我夸克开了SVIP依旧限速2M，这个问题怎么解决呢？

**Comment by 154766zyc** [comment]
既然夸克这么拉，为啥还要用夸克嘞？觉得不好用直接换别的网盘就好了啊，现在这么多家缺他？非他不可吗？不拿用户当人的商家为什么还要拥护他，给他送钱？钱多没处去可以给我分点啊喂！！！

**Comment by 123xsz** [comment]
夸克的优势是渠道会员便宜（年费个位数），离线下载比百度好使，缺点是下载慢，会员在客户端里都跑不到160mbps

在 2023年3月16日，18:49，154766zyc ***@***.***> 写道：

﻿

既然夸克这么拉，为啥还要用夸克嘞？觉得不好用直接换别的网盘就好了啊，现在这么多家缺他？非他不可吗？不拿用户当人的商家为什么还要拥护他，给他送钱？钱多没处去可以给我分点啊喂！！！

**Reply by Tenkkk** [reply]
哪里渠道个位数呀哥

**Comment by 123xsz** [comment]
咸鱼搜搜

在 2023年3月16日，22:22，Tenkkk ***@***.***> 写道：

﻿

哪里渠道个位数呀哥

**Comment by ghxtgh** [comment]
今天用不了夸克了，可以看到目录和文件，不能看视频

**Reply by beyonder588** [reply]
周末大佬们又有的忙了

**Comment by darknesscho** [comment]
今天挂载的夸克突然用不了了，我还特意更新到最新的docker容器，能读取夸克网盘里的文件列表，但视频无法播放。

**Reply by beyonder588** [reply]
周末大佬们又有的忙了

**Reply by darknesscho** [reply]
更新新版后，夸克能播放了，但速度只有100kb/s左右。。。。

**Reply by darknesscho** [reply]
补充一下，我是开了SVIP的，刚才重新获取了一下电脑浏览器的cookie，速度还是只有100kb/s。。。。。
各位又办法吗？

**Reply by xxxqqq88888888** [reply]
限速的问题解决了么

**Reply by darknesscho** [reply]
今天更新了最新版，限速解决了！

**Comment by FopFat** [comment]
夸克很慢…

**Comment by darksun199** [comment]
之前都能正常下载，今天不知为何提示：{"code":500,"message":"failed link: failed get link: download file size limit[dde801c6ec814c47977314951eaf88f1]","data":null}

**Reply by beyonder588** [reply]
下载和看视频都不行了

**Reply by wolfit50** [reply]
废了，用PC客户端的UA都不行，准备放弃了

**Reply by darksun199** [reply]
目前看，只能先夸克手机端app下载了。看看后续更新是否能解决现在的问题。要辛苦作者了。

**Reply by darksun199** [reply]
更新3.16以后，PC可以正常下载了，就是目前看速度比之前降了很多。

**Reply by wolfit50** [reply]
200K/s 🤦

**Reply by ASQ301** [reply]
我是下载有速度1.6mb/s 左右 但是观看 完全没速度太卡了

**Comment by shao60533** [comment]
Failed init storage: Get "https://drive.quark.cn/1/clouddrive/config?fr=pc&pr=ucpro": dial tcp **********56:443: connect: network is unreachable
这个怎么办

**Comment by ASQ301** [comment]
现在夸克大佬修复后 下载是有速度1.6m左右 可是无法观看太卡了  难道夸克真的废了嘛.... 难受啊

**Reply by az282661330** [reply]
同难受，夸克才是最无敌的

**Comment by az282661330** [comment]
openwrt alistv3，夸克今天访问用不成，更新了cookies和根目录还是不行，能看到文件，但是无法观看和下载，阿里云正常使用

**Reply by az282661330** [reply]
更新以后能看，但是夸克速度超慢，勉强看480p，1080p看不成。阿里云盘也变慢了，kodi整体变得卡顿

**Reply by guoyan1983** [reply]
请问如何更新？

**Reply by az282661330** [reply]
我是n1 openwrt，alist直接安装在网络存储那里。更新的话，系统，软件包，找到更新版本的alist软件安装，重启就好了

**Reply by guoyan1983** [reply]
感谢。已更新到316，希望可以看

**Reply by lamboxr** [reply]
win server 2022 , alist 3.16.3, 夸克 vip，4k无压力

**Comment by kiolygenius** [comment]
限速200KiB问题，最新提交已经可以。Docker需要用main版本，编译版的话去Github Action里面找最后一个main的编译action的artifact

**Reply by kiolygenius** [reply]
```bash
docker pull xhofe/alist:main
```
<https://github.com/alist-org/alist/actions/runs/4718633189#artifacts>

**Reply by xxxqqq88888888** [reply]
那套件版解决了限速问题了么

**Reply by kiolygenius** [reply]
套件版 是指什么？

**Reply by ASQ301** [reply]
openwrt 只能等正式版更新了嘛  现在是3.16.0

**Reply by xxxqqq88888888** [reply]
群辉nas的alist套件版本

**Reply by kiolygenius** [reply]
@xxxqqq88888888
那可能只能等版本更新了

**Reply by xxxqqq88888888** [reply]
docker pull xhofe/alist:main  这个版本要怎么下载
还是说要怎么安装使用

**Comment by LEOx2305** [comment]
夸克突然开始限速了，没法看电影。

**Comment by kayeung** [comment]
按照上述步骤，现在已经无法在线观看网盘内视频，也无法下载

**Comment by steventsang7** [comment]
使用“使用代理地址”或“本地代理”下载东西均不可以，提示“{"code":500,"message":"failed get link: download file size limit[61342449529848e8ade5444b85f641e2]","data":null}”，请问是什么意思，一直都下载不了

**Comment by Jasonanywhere** [comment]
{"code":500,"message":"failed get link: download file size limit[c76055551123432c98c2d8c399898cb8]","data":null}
为啥下载会显示这个

**Comment by marcojunior1992** [comment]
更新13.6.1 解决夸克限速问题。现在已经可以正常使用了。

**Reply by x1ao4** [reply]
请问是会员不限速了吗？我装了v3.16.3，非会员，还是限速。

**Reply by x1ao4** [reply]
13.6.1 是什么的版本？

**Reply by guoyan1983** [reply]
可以正常在线播放吗

**Comment by guoyan1983** [comment]
升级到3163，还是在线播放卡顿，有遇到的朋友吗

**Reply by guoyan1983** [reply]
设备重启后好了

**Reply by YUNNANZHOU** [reply]
重启什么？电脑？

**Reply by YoungNS** [reply]
更新完重启alist就好了

**Comment by jefaa2023** [comment]
挂载夸克无法在线和webdav方式播放视频。是没解了吗？a里云都是可以的

**Comment by looktolink** [comment]
下载限制文件大小了有办法解决吗，蛋疼。

**Comment by YUNNANZHOU** [comment]
下载慢的一比 根本用不了
![]()

**Comment by lamboxr** [comment]
夸克网盘，用tmm削刮器削刮海报，总是会先生成一个大小为0的海报图片或nfo信息，文件名形如“xxxxx-thumb.jpg”然后再生成一个实际有大小的文件，文件后缀加了个(1),甚至（2），（3），形如“xxxxx-thumb（1）.jpg” 导致kodi扫描的时候 不识别大小为0的那个文件名正确的文件

**Reply by xindongzetong** [reply]
同

**Comment by zhouyinxuan9264** [comment]
夸克网盘的列表可以加载，但是视频无法播放，重试也不行

**Comment by fgfg163** [comment]
好像夸克网盘的列表地址变了？今天忽然就提示列表获取失败，看alist请求的地址是
https://drive.quark.cn/1/clouddrive/file/sort?_fetch_total=1&_page=1&_size=100&_sort=file_type%3Aasc%2Cfile_name%3Aasc&fr=pc&pdir_fid=0&pr=ucpro
再看网页版请求的地址变成了
https://drive-pc.quark.cn/1/clouddrive/file/sort?pr=ucpro&fr=pc&pdir_fid=0&_page=1&_size=50&_fetch_total=1&_fetch_sub_dirs=0&_sort=file_type:asc,updated_at:desc

**Reply by aaabbbccc100** [reply]
打开后目录是空的，怎么解，似乎早上好一会，这时看又不行了。

**Comment by jefaa2023** [comment]
夸克吗？看阿里，百度，天翼正常的，夸克没挂载过。



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年6月5日(星期一) 凌晨1:11
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [alist-org/docs] zh/guide/drivers/quark (Discussion #20)





 
打开后目录是空的，怎么解，似乎早上好一会，这时看又不行了。

**Reply by aaabbbccc100** [reply]
打开后目录是空的，怎么解，似乎早上好一会，这时看又不行了。

**Comment by syueya** [comment]
Failed load storage: failed init storage: Get "https://drive.quark.cn/1/clouddrive/config?fr=pc&pr=ucpro": dial tcp: lookup drive.quark.cn: Try again。老是出现这个原因是为啥呀

**Comment by 12354904** [comment]
有没有现成的cookie

**Reply by Tacit-git** [reply]
大兄弟解决了没

**Comment by gwL955** [comment]
夸克好像有个问题，我登陆网页转存资源之后，alist这里就挂了要重新获取cookie

**Comment by air0451** [comment]
今天在alist通过aria2下载夸克里的文件，无论大小，下载到本地只有100多字节，为啥啊，视频也能正常播放啊！

**Comment by lbb0920** [comment]
在哪个网页上按F12打开“调试”，为啥我没有找到这个sort的

**Reply by YoungNS** [reply]
登录夸克网盘网页版按F12，在网络选项卡搜索sort

**Reply by DSQsqblxhq** [reply]
如果没有可以在 `F12` 调试页面的`网络`选项卡，输入Ctrl+R试试

**Comment by giserd** [comment]
[bug] 夸克网盘填写cookie和根目录id(全部就是0)就可以了,  文件排序不要选文件类型（默认无），选了之后会报错failed get objs: failed to list objs: query fail [61008]

**Reply by dsc-bjmn** [reply]
我去……真是这个原因，感谢老铁……

**Comment by guoyan1983** [comment]
现在用夸克在线看的话，是不是会下载内容到本机的，我是安装到电视盒子上的nplayer，过几天，盒子储存就满了。需要怎么修改设置吗

**Comment by kidgokudou** [comment]
下载都是这个问题panic: close of closed channel，怎么破

**Comment by air0451** [comment]
通过威联通docker布置alist挂载夸克成功，但是发送到aria2上无法下载，只是下载了一个一百多字节的文件，该咋办？

**Comment by onelxzy** [comment]
现在是不是不能用了，显示挂载成功，但目录都是空的，无论挂载0根目录还是具体某个子目录，列表都刷不出来东西

**Reply by 1molPotato** [reply]
我列表都能刷出来。。但下载不了也预览不了

**Comment by xingxinag** [comment]
网页版登陆入口
https://pan.quark.cn/

**Comment by dsc-bjmn** [comment]
夸克挂载之后，打开就是这样的：failed get objs: failed to list objs: query fail [61008]
有没有大佬现在能挂载成功的啊？求指点……

**Comment by 1molPotato** [comment]
求助：威联通挂载成功，也可以刷新到文件目录，但所有文件无法预览和下载，请求均返回
500 Internal Server Error。此前一直是好的，看nas日志大约是本月初开始无法访问了

**Comment by 18819740235** [comment]
有没有跟我一样，alist配合cloud sync使用，老是卡100%的，rar，pdf等格式的

**Comment by Not9Irons** [comment]
Chrome edge 登录填写的cookie 全都显示要求登录

**Reply by Not9Irons** [reply]
没事了cookie复制漏了一段

**Comment by yangjian1412** [comment]
好像不能同时挂两个不同的夸克了，会提示failed get objs: failed to list objs: token [st invalid,code:50051]

**Comment by liquidsnake9** [comment]
挂载成功，能上传能下载，但emby的strm链接，缺无法播放，挂载的其他几个网盘播放都没问题，是因为夸克强制用的本地代理的原因嘛？

**Comment by red-blakTree** [comment]
夸克不开会员,将端口号设置为6800速度可以达到2mb/s,服务器最大连接数为64

**Reply by xurdy** [reply]
你好，请问如何设置？

**Comment by ZhangRuoYang666** [comment]
实在排查不出原因，还请官方大大看下。
挂载夸克后，有些视频可以观看，有些观看不了（验证过和视频大小无关）。举例文件视频地址
链接：https://pan.quark.cn/s/6382d9afaac2；跟踪log文件，找到如下报错信息：

ServeHttp error. err: write tcp [::1]:5244->[::1]:61408: wsasend: An established connection was aborted by the software in your host machine.  
[33mWARN[0m[2024-08-16 19:19:19] Maybe size incorrect or reader not giving correct/full data, or connection closed before finish. written bytes: 3533 ,sendSize:50618143,

**Comment by jinbo0400** [comment]
进入夸克目录都要转圈，播放视频（下载）时刚开始都要卡一下，有同样的吗？

**Comment by icexk** [comment]
现在没有夸克TV驱动了吗

**Reply by JJJJCG** [reply]
20号更新文档刚加上的，应该是还没上线

**Reply by icexk** [reply]
好的

**Reply by lm379** [reply]
beta版已经上线了

**Reply by icexk** [reply]
beta版，按文档还用不起

**Comment by Sanmmz** [comment]
为啥最新得alist还是没有夸克tv驱动

**Comment by hmgdxfoss** [comment]
老板，夸克TV能否默认按照文件名排序。目前的是默认按照修改时间排序，导致文件很乱。且存储设置里没有按照文件名排序的设置

**Comment by AutisticY** [comment]
Get "https://drive.quark.cn/1/clouddrive/config?fr=pc&pr=ucpro": dial tcp [******************]:443: connect: permission denied|
为什么提示被禁止了

**Reply by AutisticY** [reply]
被防火墙禁了？

**Reply by AutisticY** [reply]
其他网盘是可以的

**Comment by okey2k** [comment]
Failed init storage: Get "https://drive.quark.cn/1/clouddrive/config?fr=pc&pr=ucpro": dial tcp **********3:443: connect: no route to host

连接的夸克网盘，一直正常，今天突然出这个，更新了cookie也没用，有知道处理方法的大神在么？

**Comment by hmgdxfoss** [comment]
老板，夸克TV能不能默认按照文件名排序，现在默认修改时间排序导致文件很乱，希望能修复

**Comment by Hawkli0777** [comment]
请问为什么夸克tv驱动302，比夸克驱动播放速度慢好多啊？uctv302，也比uc驱动慢

**Reply by cuibo365** [reply]
同问，会员也只有200K每秒

**Comment by SunRain** [comment]
修改aria2的user-agent为手机浏览器的UA似乎可以增加夸克网盘的下载速度.
测试环境如下:
```
电信千兆宽带
淘宝88vip赠送的夸克会员
ArchLinux + kde桌面环境
使用单独的Docker aria2(非AList集成aria2的版本),并设置aria2的UA为小米mix4的手机UA,
AList后台'设置=>其他'下添加上述aria2的地址和密钥
测试1:AList前端选择一个文件,点击下载,使用kde自带的下载功能,网速可以达到10M网速
测试2:AList前端选择一个文件链接,复制到AriaNG下载(AriaNG同样设置UA为手机的UA),平均大概20-30M网速



```

**Comment by MCWXT** [comment]
Failed init storage: require login [guest]

其实这个是什么意思？

**Comment by Hawkli0777** [comment]
谢谢🙏




--
发自我的网易邮箱手机智能版



在 2024-09-30 17:12:31，SunRain ***@***.***> 写道：


修改aria2的user-agent为手机浏览器的UA似乎可以增加夸克网盘的下载速度.
测试环境如下:

电信千兆宽带
淘宝88vip赠送的夸克会员
ArchLinux + kde桌面环境
使用单独的Docker aria2(非AList集成aria2的版本),并设置aria2的UA为小米mix4的手机UA,
AList后台'设置=>其他'下添加上述aria2的地址和密钥
测试1:AList前端选择一个文件,点击下载,使用kde自带的下载功能,网速可以达到10M网速
测试2:AList前端选择一个文件链接,复制到AriaNG下载(AriaNG同样设置UA为手机的UA),平均大概20-30M网速
